pkg/handlers/dev/start.go

```diff
@@ -89,7 +89,7 @@ var (
 		"-p", "8484:8484",
 		"-p", "9292:9292",
 		"-v", "$ASERTO_CERTS_DIR:/certs:rw",
-		"-v", "$ASERTO_CFG_DIR:/app/cfg:ro",
+		"-v", "$ASERTO_CFG_DIR:/cfg:ro",
 		"-v", "$ASERTO_EDS_DIR:/app/db:rw",
```

Comment:
Should the `"-v", "$ASERTO_EDS_DIR:/app/db:rw",` also be updated to remove the `app` part